src/terra-dev-site/snt/secondary scroll test/parent-folder/child-folder/example.snt.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
```

Comment:
can we remove the spaces in these file-paths and use `-` instead?

Reply:
Ok done: https://github.com/cerner/terra-dev-site/pull/217/commits/ca7838a20d7c4ba956d97f84426acd2cb05cd6f4